test/production/app-dir/metadata-streaming-config/metadata-streaming-config.test.ts

```diff
@@ -9,23 +9,34 @@ describe('app-dir - metadata-streaming-config', () => {
     const requiredServerFiles = JSON.parse(
       await next.readFile('.next/required-server-files.json')
     )
-    expect(requiredServerFiles.files).toContain(
-      '.next/response-config-manifest.json'
-    )
+
     expect(
       requiredServerFiles.config.experimental.htmlLimitedBots
     ).toMatchInlineSnapshot(
       `"Mediapartners-Google|Slurp|DuckDuckBot|baiduspider|yandex|sogou|bitlybot|tumblr|vkShare|quora link preview|redditbot|ia_archiver|Bingbot|BingPreview|applebot|facebookexternalhit|facebookcatalog|Twitterbot|LinkedInBot|Slackbot|Discordbot|WhatsApp|SkypeUriPreview"`
     )
 
-    const responseConfigManifest = JSON.parse(
-      await next.readFile('.next/response-config-manifest.json')
+    const prerenderManifest = JSON.parse(
+      await next.readFile('.next/prerender-manifest.json')
     )
+    const { routes } = prerenderManifest
+
+    const bypassConfigs = Object.keys(routes)
+      // Pick the 3rd bypass config for each route
+      .map((route) => [route, routes[route].experimentalBypassFor?.[2]])
```

Comment:
this seems racey -- why pick the third rather than just filtering the bypass configs that match on `user-agent`?